DOCUMENTATION.md

```diff
@@ -638,8 +638,10 @@ The following is the list of all client properties in alphabetical order.
-| hazelcast.client.statistics.period.seconds             | 3                                   | number          | Period in seconds the client statistics are collected and sent to the cluster.                                                                                                                                                                                                                                                                                                                                      |
+| hazelcast.client.statistics.enabled                    | false                               | boolean         | **DEPRECATED since 5.1, use `hazelcast.client.metrics.enabled` instead** If set to `true`, it enables collecting the client statistics and sending them to the cluster. When it is `true` you can monitor the clients that are connected to your Hazelcast cluster, using Hazelcast Management Center.                                                                                                              |
+| hazelcast.client.statistics.period.seconds             | 5                                   | number          | **DEPRECATED since 5.1, use `hazelcast.client.metrics.collection.frequency` instead** Period in seconds the client statistics are collected and sent to the cluster. Must be positive.                                                                                                                                                                                                                              |
+| hazelcast.client.metrics.enabled                       | false                               | boolean         | If set to `true`, it enables collecting the client metrics and sending them to the cluster. When it is `true` you can monitor the clients that are connected to your Hazelcast cluster, using Hazelcast Management Center.                                                                                                                                                                                          |
+| hazelcast.client.metrics.collection.frequency          | 5                                   | number          | Period in seconds the client metrics are collected and sent to the cluster. Must be positive.                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Should `metrics` be a configuration section instead of properties?

Reply:
It could be. It may be easier to configure like that. @mdumandag WDYT?

If it will be a separate section, it will be a top level config I guess. Like this:
```js
{
      metrics: {
         enabled: true,
         frequency: 3
      }
}
```

Reply:
Making it top level makes sense, it's like that in the Java client and the Go client (there we use a different name).

Reply:
Yeah working on it, almost done thanks for the heads up.